### DISCUSSION TITLE
Add auto-linking to GitHub repository configuration

### DISCUSSION THREAD

**Discussion by lahma** [ravendb/ravendb#16325] [Ideas]
Would it make sense for you to add autolink configuration to this repository so issue keys become clickable and one can then navigate to issue tracker using them. Would be a handy shorthand.

* Numeric
* Reference prefix: `RavenDB-`
* Target URL: `https://issues.hibernatingrhinos.com/issue/RavenDB-<num>`

![image](https://user-images.githubusercontent.com/171892/233666391-92e3eb4a-5701-491d-84af-f0cb074bc1fb.png)

**Comment by ppekrol** [comment]
Thanks @lahma 
We will look into that.
